syntax="proto3";
package tinyrpc;
message RpcHeader
{
    bytes service_name=1;
    bytes method_name=2;
    bytes method_args=3;
}